ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -71,6 +71,12 @@ releases:
       product: ocp
       stream: nightly
       version: "4.13"
+  source:
```

Comment:
openshift-e2e-test-qe does not reference openshift-extended-upgrade-pre, so we do not need the setting here.

Reply:
remove it, thanks